hack/lib/catalogsource.bash

```diff
@@ -64,16 +64,6 @@ spec:
         - containerPort: 50051
           name: grpc
           protocol: TCP
-        livenessProbe:
-          exec:
-            command:
-            - grpc_health_probe
-            - -addr=localhost:50051
-        readinessProbe:
-          exec:
-            command:
-            - grpc_health_probe
-            - -addr=localhost:50051
```

Comment:
Please keep this, we only need to drop the livenessProbe I believe as that'll initiate a premature restart.

Reply:
I didnt notice that but I guess it could be the case. I will get it back.Btw I tried some numbers to set the initial delay there but didnt help in all cases.